Project.toml

```diff
@@ -13,7 +13,6 @@ Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
-KrylovKit = "0b1a1467-8014-51b9-945f-bf0ae24f4b77"
```

Comment:
Ah, I did not quite catch that we *already* depend on KrylovKit. We should probably not do that, but you have to leave it for now or the examples will break